Dfe.Data.SearchPrototype/Web/Tests/PageIntegrationTests/SearchResultsPageTests.cs

```diff
@@ -28,13 +31,21 @@ public HomePageTests(PageWebApplicationFactory<Program> factory, ITestOutputHelp
         }
 
         [Fact]
-        public async Task Search_Title_IsDisplayed()
```

Comment:
Leaking concerns 

`HtmlHelpers.GetDocumentAsync(response)` is a concern for how a QueryClient gets its dependency to query a document.

`document.GetElementText` is an AngleSharp concern for how it queries the DOM.

`HomePage.Heading.Criteria` is a WebDriver type that leaks through to any test using the pages.